dotCMS/src/main/webapp/html/js/dotcms/dijit/image/ImageEditor.js

```diff
@@ -2,1174 +2,1088 @@ dojo.provide("dotcms.dijit.image.ImageEditor");
+				ctx.iframe.dojo.byId("displayImageHeight").value = sic.h;
+				ctx.baseAndShowLoaded();
+			};
+		}
+		dojo.attr(this.iframe.dojo.byId("imageViewPort"), "class", "");
+	},
+
+	baseAndShowLoaded: function () {
+		var showImage = this.iframe.dojo.byId("me");
+		var baseImage = this.iframe.dojo.byId("baseImage");
+
+		if (showImage.complete && baseImage.complete && !this.imagesLoaded) {
+			this.imagesLoaded = true;
+
+			var sic = dojo.coords(showImage);
+			var bic = dojo.coords(baseImage);
+
+			var x = Math.round((sic.w / bic.w) * 100);
+			this.zoomValue = 0;
+			if (!isNaN(x)) {
+				this.zoomValue = x;
+			}
+
+			var x = this.iframe.dijit.byId("showScaleSlider");
+
+			x.attr("value", this.zoomValue);
+			this._updateZoomFactor();
+		}
+	},
+
+	closeImageWindow: function (e) {
+		this._cleanUpImageEditor();
+	},
+
+	/**
+	 * cleans up old references, resets imageEditor
+	 */
+	_cleanUpImageEditor: function (e) {
+		window.top._dotImageEditor = null;
+		window.parent._dotImageEditor = null;
+
+		var myEditor = this.imageEditor
+			? this.imageEditor
+			: document.getElementById("dotImageDialog");
+
+		// if we have an Image Editor
+		if (myEditor) {
+			while (this.imageEditor.firstChild) {
+				this.imageEditor.removeChild(this.imageEditor.firstChild);
+			}
+			myEditor.parentNode.removeChild(myEditor);
+		}
+
+		this.imageEditor = null;
+		this.iframe = null;
+		this.inited = false;
+		this.painting = false;
+		this.filters = new Array();
+		this.saveAsIncrement = 1;
+		this.imagesLoaded = false;
+		this.resizeFilter = false;
+	},
+
+	/***************************************************************************
+	 *
+	 * Button Actions
+	 *
+	 **************************************************************************/
+
+	addToClipboard: function () {
+		if (this.currentUrl.indexOf("/temp_") > -1) {
+			alert(
+				"You cannot clip a temp or altered file.  Please save the content and reopen the image editor to clip it"
+			);
+			return;
+		}
+		var fileUrl = this.cleanUrl(this.currentUrl);
+		var url =
+			fileUrl.indexOf("?") > -1
+				? fileUrl + "&"
+				: fileUrl + "?_imageToolClipboard=true";
+		//alert(url);
+		var target = this.iframe.dojo.byId("me");
+		dojo.style(target, "opacity", 0);
+		dojo.xhrGet({
+			url: url,
+			target: target,
+			preventCache: true,
+			sync: true,
+			load: function (data) {
+				dojo.fadeIn({ node: target }).play();
+			},
+			error: function (data) {
+				dojo.style(target, "opacity", 100);
+			},
+		});
+	},
+
+	/**
+	 * Save as passes a var:_imageToolSaveFile to the binary servlet with all the other
+	 * params.  This saves the file handle in the users session.  We look for this
+	 * file and save it when the user checks the file or content in
+	 */
+
+	saveImage: function () {
+		if (this.binaryFieldId != null && this.binaryFieldId.length > 0) {
+			this.saveBinaryImage(this.activeEditor);
+		} else {
+			this.saveFileImage();
+		}
+	},
+
+	_isValidURL: function (url) {
```

Comment:
Added this helper for a quick check in `saveBinaryImage`, the reason for this is that the URL comes in either host + path or just path, so we need to count for that.